apm-agent-plugins/apm-jdbc-plugin/src/main/java/co/elastic/apm/agent/jdbc/ConnectionInstrumentation.java

```diff
@@ -50,9 +51,12 @@
 public class ConnectionInstrumentation extends JdbcInstrumentation {
 
     @Advice.OnMethodExit(suppress = Throwable.class, inline = false)
-    public static void storeSql(@Advice.Return PreparedStatement statement,
+    public static void storeSql(@Advice.Return @Nullable PreparedStatement statement,
                                 @Advice.Argument(0) String sql) {
-        getJdbcHelper().mapStatementToSql(statement, sql);
```

Comment:
fixed potential NPE when statement is null (as map does not allow for null values)